cargo-miri/bin.rs

```diff
@@ -6,7 +6,7 @@ use std::io::{self, BufRead, BufReader, BufWriter, Read, Write};
 use std::iter::TakeWhile;
 use std::ops::Not;
 use std::path::{Path, PathBuf};
-use std::process::Command;
+use std::process::{Command, Stdio};
```

Comment:
I think it'd be better to just import `process` and use `process::Stdio` explicitly -- the type is not used that much and the name requires some context to make sense.

(Note there already are used of `std::process::Stdio` in this file, those can then become `process::Stdio`.)